File: Periodic Table/NobleGas.py
'''
Created on Dec 6, 2013

@author: thomas
'''

from NonMetal import NonMetal

class NobleGas(NonMetal):
    
    REACTION_LEVEL = 0

    CHARGE = 0

    def __init__(self, atomicNumber, atomicMass):
        self.atomicNumber = atomicNumber
        self.atomicMass = atomicMass

neon = NobleGas(10, 20.179)
argon = NobleGas(18, 39.948)
krypton = NobleGas(36, 83.8)
xenon = NobleGas(54, 131.29)
radon = NobleGas(86, 222)

File: Java/src/Table/NobleGas.java
package Table;

/**
 * Created by Thomas on 1/31/14.
 */
public class NobleGas extends NonMetal{

    int atomicNumber;
    double atomicMass;

    NobleGas(int aN, double aM){
        int atomicNumber = aN ;
        double atomicMass = aM;
    }

    NobleGas helium = new NobleGas(2, 4.0026);
    NobleGas neon = new NobleGas(10, 20.179);
    NobleGas argon = new NobleGas(18, 39.948);
    NobleGas krypton = new NobleGas(36, 83.8);
    NobleGas radon = new NobleGas(86, 222);

}
